packages/picasso/src/Notes/Notes.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { forwardRef } from 'react'
+import { makeStyles, Theme } from '@material-ui/core/styles'
+import { BaseProps, CompoundedComponentWithRef } from '@toptal/picasso-shared'
+import cx from 'classnames'
+
+import NotesTitle from '../NotesTitle'
+import NotesSubtitle from '../NotesSubtitle'
+import NotesContent from '../NotesContent'
+import styles from './styles'
+import Container from '../Container'
+
+export interface Props extends BaseProps {
+  children: React.ReactNode
+}
+
+const useStyles = makeStyles<Theme>(styles)
+
+export interface StaticProps {
+  Title: typeof NotesTitle
+  Subtitle: typeof NotesSubtitle
+  Content: typeof NotesContent
+}
+
+export const Notes = forwardRef<HTMLDivElement, Props>(function Notes (
```

Comment:
In the story's Acceptance Criteria I see this component should be called `Note`.

```suggestion
export const Note = forwardRef<HTMLDivElement, Props>(function Note (
```